src/main/java/org/swisspush/redisques/util/QueueConfiguration.java

```diff
@@ -0,0 +1,71 @@
+package org.swisspush.redisques.util;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+import java.util.List;
+
+public class QueueConfiguration {
+    
+    public static final String PROP_PATTERN = "pattern";
+    public static final String PROP_RETRY_INTERVALS = "retryIntervals";
+    
+    private String pattern;
+    
+    private List<Integer> retryIntervals;
+
+    public String getPattern() {
+        return pattern;
+    }
+
+    public List<Integer> getRetryIntervals() {
+        return retryIntervals;
+    }
+
+    public static QueueConfigurationBulider with() {
+        return new QueueConfigurationBulider();
+    }
+    
+    public JsonObject asJsonObject() {
+        JsonObject jsonObject = new JsonObject();
+        jsonObject.put(PROP_PATTERN, getPattern());
+        jsonObject.put(PROP_RETRY_INTERVALS, new JsonArray(getRetryIntervals()));
+        return jsonObject;
+    }
+    
+    public static QueueConfiguration fromJsonObject(JsonObject jsonObject) {
+        QueueConfigurationBulider builder = QueueConfiguration.with();
+        if (jsonObject.containsKey(PROP_PATTERN)) {
+            builder.pattern(jsonObject.getString(PROP_PATTERN));
+        }
+        if (jsonObject.containsKey(PROP_RETRY_INTERVALS)){
+            builder.retryIntervals(jsonObject.getJsonArray(PROP_RETRY_INTERVALS).getList());
+        }
+        return builder.build();
+    }
+
+    private QueueConfiguration(QueueConfigurationBulider builder) {
+        this.pattern = builder.pattern;
+        this.retryIntervals = builder.retryIntervals;
+    }
+    
+    public static class QueueConfigurationBulider {
```

Comment:
I would avoid using a `...Builder` class. I personally see no utility for it.
Just having a simple, Configuration POJO/Bean (with fluent setters) is sufficient. 
Or can you explain the utility for having such a `...Builder`?

Reply:
We also have a RedisquesConfigurationBuilder so we might keep it for consistency reasons?